src/Question.js

```diff
@@ -1,49 +0,0 @@
-import React from "react";
-import Header from "./Header";
-
-const Question = ({ quizQuestions, quizOptions, images, setQuesAnswered, callBack, id, choice }) => {
-
-    return (
-        <>
-            <Header />
-            <div className="container">
-                <div className="heading2Div">
-                    <h2> {quizQuestions} </h2>
-                </div>
-                <div className="options" >
-                    {
-                        quizOptions.map((value) =>
-                        (<>
-                            <div className="selectOptions">
-                                <input key={value}
-                                    type="radio"
-                                    id={id}
-                                    name="radioBtn"
-                                    value={value}
-                                    onChange={
-                                        () => { callBack(value); console.log(value); }}
-                                />
-                                <label htmlFor="mood"
-                                    key={value.toUpperCase()}
-                                > {value} </label>
-                            </div>
-                        </>
-                        )
-                        )
-                    }
-                </div>
-                <div className="imgDiv" >
-                    <img className="mainImg" src={images} alt="" />
```

Comment:
nit: how about we name this `imgContent`?

Reply:
Changed :)